config/prow/cluster/ghproxy.yaml

```diff
@@ -64,6 +64,16 @@ spec:
         volumeMounts:
         - name: cache
           mountPath: /cache
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 3
+          periodSeconds: 3
+        readinessProbe:
```

Comment:
Do we need to specify these fields for the `readinessProbe` like we do elsewhere? https://github.com/kubernetes/test-infra/blob/c2295bcc10781ae4835f6289f4f822e99982751c/config/prow/cluster/hook_deployment.yaml#L92-L94

Reply:
Addressed